doc/usage/installation.rst

```diff
@@ -2,30 +2,86 @@
 Installing Sphinx
 =================
 
-.. contents::
-   :depth: 1
+Sphinx is a Python application. It can be installed in one of the ways described
+below.
+
+.. contents:: Installation methods
+   :depth: 2
    :local:
    :backlinks: none
 
 .. highlight:: console
 
-Overview
---------
+After installation, you can check that Sphinx is available by running ::
+
+   $ sphinx-build --version
+
+This should print out the Sphinx version number.
+
+
+.. tip::
+
+   If you use Sphinx for documenting a Python library or application, it is
+   generally recommended to install Sphinx into your development environment
+   (`venv <https://docs.python.org/3/library/venv.html>`_ or `conda
+   <https://conda.io/projects/conda/en/latest/user-guide/getting-started.html>`_
+   environment).
+
+   By adding Sphinx and 3rdparty extensions or themes that you use to your dev
+   dependencies, you make sure that you have a consistent setup for building
+   your documentation.
+
+
+.. _install-pypi:
+
+PyPI package
+------------
+
+Sphinx packages are published on the `Python Package Index
+<https://pypi.org/project/Sphinx/>`_ (PyPI).  The preferred tool for installing
+packages from *PyPI* is :command:`pip`, which is included in all modern versions of
+Python.
+
+Run the following command::
+
+   $ pip install -U sphinx
+
```

Comment:
```suggestion

.. tip::

   To avoid issues when rebuilding your environment,
   it is advisable to pin sphinx and third-party extension
   versions in a `requirements.txt file`__::
   
      $ pip install -r requirements.txt
      
   Or, if writing documentation for a Python package,
   place the dependencies in the `pyproject.toml file`__::
   
      $ pip install .[docs]

   __ https://pip.pypa.io/en/stable/reference/requirements-file-format/
   __ https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-optional-dependencies

```